packages/rancher-vsphere/rancher-vsphere-cpi/package.yaml

```diff
@@ -1,4 +1,4 @@
 url: https://github.com/rancher/vsphere-charts.git
 subdirectory: charts/rancher-vsphere-cpi
-commit: 27242a6495e33d34286d557459eb3cab785e26dd
-version: 100.1.1
+commit: 81da645ddf86370fc5d1e35dc4711a45b3cea929
+version: 100.2.0
```

Comment:
from @PennyScissors:

>we should bump the version to 100.2.0 because 1.1.0 and 2.4.1-rancher1 introduces support for a new k8s version. Aside from that lgtm (e